### ISSUE TITLE
Do not include all moment locales in JS bundle

### ISSUE THREAD

**Issue by jseppi** [18F/federalist#1037]
From `create-react-app`'s webpack config:

```js
// Moment.js is an extremely popular library that bundles large locale files
// by default due to how Webpack interprets its code. This is a practical
// solution that requires the user to opt into importing specific locales.
// https://github.com/jmblog/how-to-optimize-momentjs-with-webpack
// You can remove this if you don't use Moment.js:
new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Essentially, we have probably ~150Kb of extra stuff in our bundle.

**Comment by jseppi** [comment]
Might also want to try out the `webpack-bundle-analyzer`: https://www.npmjs.com/package/webpack-bundle-analyzer

**Comment by wslack** [comment]
why was this closed?

**Comment by wslack** [comment]
hypothesis: was fixed in an unlinked PR

**Comment by jseppi** [comment]
It was fixed in #1038, which was connected via ZenHub.

**Comment by wslack** [comment]
my mistake, wrong laptop
